Modules/SubModule_Import/xASL_wrp_NII2BIDS_Subject.m

```diff
@@ -39,7 +39,7 @@
     % Subsequent code is based on having data per ASL scan, so we "fool" it
     % by renaming all runs into ASL_1 ASL_2 ASL_n and keeping the
     % unique runs only. Missing scans will issue a warning, not an error.
-    listRuns = xASL_adm_GetFileList(fullfile(x.modules.import.imPar.TempRoot,nameSubjectSession),'^(ASL|T1w|FLAIR).+$',false,[],true);
+    listRuns = xASL_adm_GetFileList(fullfile(x.modules.import.imPar.TempRoot,nameSubjectSession),'^(ASL|T1|T2|FLAIR).+$',false,[],true);
```

Comment:
are you sure that this is a good idea?

Reply:
Well, this way, both T1w and T1 are supported. And T2 as well.
I would think it is the best if the first `dcm2nii` is very flexible, and the subsequent `nii2bids` should be restricted (and we can then solve problems in this stage).

Reply:
You have convinced me - see the other comment that helps to fix this. Cause we have predefined inputs and assigned outputs. Enough to change this config file to make sure that things are solved on the `nii2bids` side.

Reply:
Which other comment?